bugnote_view_inc.php

```diff
@@ -137,14 +141,13 @@
 	$t_normal_date_format = config_get( 'normal_date_format' );
 	$t_total_time = 0;
 
-	# Tokens for action buttons are created only once, if needed
-	$t_security_token_state = null;
-	$t_security_token_notes_delete = null;
-	$t_security_token_attachments_delete = null;
-
 	for( $i=0; $i < $t_activities_count; $i++ ) {
 		$t_activity = $t_activities[$i];
 
+		if( $t_activity['type'] === ENTRY_TYPE_ATTACHMENT ) {
```

Comment:
I know we currently only have 2 activity types at the moment, but shouldn't this be `!== ENTRY_TYPE_NOTE` ?